collect_app/src/main/res/drawable/main_menu_button_background.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
+    android:color="@color/color_on_surface_low_emphasis">
+    <item>
+        <shape android:shape="rectangle">
+            <corners android:radius="100dp" />
+            <solid android:color="@color/colorSurfaceContainer" />
```

Comment:
This should really reference a theme attribute rather than a color directly. Just looking at the [Elevated button spec](https://m3.material.io/components/buttons/specs#0eea2a85-b4d7-4c74-b08e-98410b9412c7) again, it looks like the color token being used is referred to is "surface container low" (see the Container color). 

I was super confused as to why the attribute doesn't exist already, given it's documented [here](https://github.com/material-components/material-components-android/blob/master/docs/theming/Color.md), but it looks like it's just not in a release yet (it's in [1.10.0-alpha01 currently](https://github.com/material-components/material-components-android/releases/tag/1.10.0-alpha01)). I vote that we create a custom attribute in `attrs.xml` for the moment and use that until we can upgrade to a stable version of Material Components with the full set of Material color attributes. We could add a special block for polyfilling attributes like this:

```xml
<!--  Should be removed when these attributes are added to Material Components  -->
<declare-styleable name="Material3PolyFill">
        <attr name="colorSurfaceContainerLow" format="color" />
</declare-styleable>
```

Then we can just define our `colorSurfaceContainerLow` in `theme.xml` and kill the polyfill block when we upgrade.

Reply:
Done.